draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx

```diff
@@ -0,0 +1,127 @@
+   * an initial animation.
+   */
+  ambientAnimation: string
+}
+
+/**
+ * TODO
+ * ambient and initial animation
+ * convert to mp4 for safari
+ */
+export const VideoPlayer = ({
+  autoplay = true,
+  loop = false,
+  fallback,
+  ambientAnimation,
+}: VideoPlayerProps) => {
+  const videoRef = useRef<HTMLVideoElement>(null)
+  const reducedMotionQuery = window.matchMedia(
+    "(prefers-reduced-motion: reduce)"
+  )
+  const [
+    prefersReducedMotion,
+    setPrefersReducedMotion,
+  ] = React.useState<boolean>(reducedMotionQuery.matches || false)
+
+  useEffect(() => {
+    /**
+     * Setting `muted` on the player is required in Safari/Webkit and older
+     * versions of Chome for the video to autoplay (regardless of whether
+     * the format contains an audio stream or not).
+     *
+     * React does not guarentee the `muted` attribute is set on the
+     * `video` element. So on load we force set this attribute. See issue:
+     * https://github.com/facebook/react/issues/10389
+     */
+    const { current: videoElement } = videoRef
+    if (videoElement !== null) {
+      videoElement.setAttribute("muted", "")
+    }
+  }, [])
+
+  useEffect(() => {
+    if (!reducedMotionQuery.addEventListener || !window) return
+    const updateMotionPreferences = () => {
+      const { matches = false } = window.matchMedia(
+        "(prefers-reduced-motion: reduce)"
+      )
+      setPrefersReducedMotion(matches)
+    }
+    reducedMotionQuery.addEventListener("change", updateMotionPreferences, true)
+
+    // eslint-disable-next-line prefer-arrow/prefer-arrow-functions
+    return function cleanup() {
+      reducedMotionQuery.removeEventListener("change", updateMotionPreferences)
+    }
+  }, [])
+
+  useEffect(() => {
+    const { current: videoElement } = videoRef
+    if (!videoElement) return
+    if (prefersReducedMotion) {
+      videoElement.pause()
+    } else if (autoplay && !prefersReducedMotion) {
+      videoElement.play().catch(err =>
+        /*
+         * An DOMException _may_ be raised by some browsers if we
+         * programatically interact with the video before the
+         * user has interacted with the page. This is okay - so
+         * we're going to catch this error without handling it. See:
+         * https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide#autoplay_availability
+         */
+        // eslint-disable-next-line no-console
+        console.warn(err)
+      )
+    }
+  }, [prefersReducedMotion])
+
+  return (
+    <video
+      aria-hidden={true}
+      preload="metadata"
+      ref={videoRef}
+      width="100%"
+      data-testid="kz-video-player"
+      className={styles.wrapper}
+      loop={loop}
+      poster={assetUrl(fallback)}
+      autoPlay={prefersReducedMotion ? false : autoplay}
+      playsInline={true}
+    >
+      <source src={assetUrl(ambientAnimation)} type="video/webm" />
+      Sorry, your browser doesn't support embedded videos.
```

Comment:
Not a big deal but I wonder if displaying nothing is a better fallback here, given the illustration is just for presentational purposes.

Reply:
Yeah - agreed. I'll fix this up.